Sources/TuistLoader/Loaders/ManifestLoaderFactory.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import TuistSupport
+
+public final class ManifestLoaderFactory {
+    private let useCache: Bool
+    public convenience init() {
+        let cacheSetting = Environment.shared.tuistVariables["TUIST_CACHE_MANIFESTS"] ?? "1"
```

Comment:
What about moving the `"TUIST_CACHE_MANIFESTS"` to the `Constants.swift` so that we can see at a glance which environment variables Tuist is using.